app/controllers/sms_controller.rb

```diff
@@ -75,9 +75,14 @@ def deliver_reply(reply)
       if incoming_adapter.reply_style == :via_adapter
         outgoing_adapter.deliver(reply)
       else # reply via response
-        incoming_adapter.prepare_message_for_delivery(reply)
+        begin
+          raise ENV["STUB_REPLY_ERROR"].constantize if Rails.env.test? && ENV["STUB_REPLY_ERROR"].present?
+          incoming_adapter.prepare_message_for_delivery(reply)
+        rescue StandardError
```

Comment:
It's generally not good to rescue `StandardError`. Can you be more specific about the type of error you want to catch? We know that the adapter may be returning `Sms::Error`. Maybe catch that one?